src/client/Inode.cc

```diff
@@ -274,9 +279,12 @@ bool Inode::caps_issued_mask(unsigned mask, bool allow_impl)
     // bah.. touch them all
     for (auto &pair : caps) {
       pair.second.touch();
+      session->cap_hit();
     }
     return true;
   }
+
+  session->cap_miss();
```

Comment:
For the miss counter, it is very possible that it only always counted in the first session we get in get_session().

Maybe we should let the cap hit/miss as global metric just like the read/write/metadata and dentry lease hit/miss in kclient do.

Will it make sense ?